pkgs/servers/nextcloud/packages/apps/hmr_enabler.nix

```diff
@@ -0,0 +1,35 @@
+{
+  php,
+  fetchFromGitHub,
+  lib,
+}:
+
+php.buildComposerProject (finalAttrs: {
+  pname = "hmr_enabler";
+  version = "0-unstable-2024-08-24";
+
+  src = fetchFromGitHub {
+    owner = "nextcloud";
+    repo = "hmr_enabler";
+    rev = "d5d9d330d405ac4aa0de1a87d1133784560462ed";
+    hash = "sha256-uLCpwvMVQ20z9vlO5q/GVPnaaQZ7ZjE8+V/zuqaB9Yo=";
+  };
+
+  composerNoDev = false;
+
+  vendorHash = "sha256-ZuEEcFT+q49CCooEwdiu2Co345s0ZCC7jeEksi6A99A=";
+
+  postInstall = ''
+    cp -r $out/share/php/hmr_enabler/* $out/
```

Comment:
Is `buildComposerProject` always installing into `/share/...`? In that case you may also override `installPhase` itself, undoing the installPhase in postInstall seems kinda ugly to me.